lib/ioc-rc

```diff
@@ -555,7 +557,8 @@ __rc_jails () {
         local boot_order=$(sort -n $boot_list)
         local shutdown_order=$(sort -rn $boot_list)
     else
-        __info "no jails found with property boot=on, exiting.."
+        __info "no jails found with property boot=on or
+        with jail_hostid: $hostid on this host, exiting.."
```

Comment:
Sorry I wasn't thinking straight. It's been "that" kind of a Monday. Change line 560 to:
```
__info "no jails found with property boot=on or
```
and line 561 to:
```
echo "  with jail_hostid: $hostid on this host, exiting."
```